src/main/resources/application.yml

```diff
@@ -38,78 +38,92 @@ logging:
 
 cx-flow:
   # Agreed upon shared API token
-  token: xxxx
-  bug-tracker: JIRA
+  token: fastcat
```

Comment:
Hi Ken, is it possible to leave the token value xxxx ?
Same comment for line 122

Reply:
No problem. Did not mean to introduce this.